AppConnectSwift/CreateAccountViewController.swift

```diff
@@ -45,8 +46,15 @@ class CreateAccountViewController: UIViewController {
     }
     
     func showAlert(title: String, message: String) {
+        var okAction = UIAlertAction(title: "OK", style: UIAlertActionStyle.Default) {
+            UIAlertAction in
+            NSOperationQueue.mainQueue().addOperationWithBlock({
+                self.performSegueWithIdentifier("CreateAccountSuccess", sender: nil)
+            })
+        }
```

Comment:
I think rather than the showAlert having special logic to go back to the home screen, maybe it could accept a custom completion handler?